baselines/webworker.generated.d.ts

```diff
@@ -172,22 +202,28 @@ declare var Blob: {
     new (blobParts?: any[], options?: BlobPropertyBag): Blob;
 };
 
+interface BlobPropertyBag {
+    endings?: string;
+    type?: string;
+}
+
 interface Body {
     readonly bodyUsed: boolean;
     arrayBuffer(): Promise<ArrayBuffer>;
     blob(): Promise<Blob>;
+    formData(): Promise<FormData>;
     json(): Promise<any>;
     text(): Promise<string>;
 }
 
 interface Cache {
-    add(request: RequestInfo): Promise<void>;
-    addAll(requests: RequestInfo[]): Promise<void>;
-    delete(request: RequestInfo, options?: CacheQueryOptions): Promise<boolean>;
-    keys(request?: RequestInfo, options?: CacheQueryOptions): Promise<Request[]>;
-    match(request: RequestInfo, options?: CacheQueryOptions): Promise<Response>;
-    matchAll(request?: RequestInfo, options?: CacheQueryOptions): Promise<Response[]>;
-    put(request: RequestInfo, response: Response): Promise<void>;
+    add(request: Request | string): Promise<void>;
```

Comment:
It seems like `RequestInfo` became `Request | string` (does `RequestInfo` still exist?). Is this a break we should avoid?

Reply:
We used to serialize the `type-original` property in the past, and now we use `type`. `type-original` used to have ` or ` in them and `<>` for type arguments, etc.. `type` is more rich, e.g. an array in case of union, and type argumetns being separate, etc.. but they do not use type aliases..

long story short, i will add the type aliases again.

Reply:
Added the aliases back